packages/hash/api/src/graphql/createApolloServer.ts

```diff
@@ -126,13 +130,11 @@ export const createApolloServer = ({
           };
         },
       },
+      process.env.NODE_ENV === "production"
+        ? ApolloServerPluginLandingPageDisabled()
+        : ApolloServerPluginLandingPageGraphQLPlayground({
+            settings: { "request.credentials": "include" },
+          }),
```

Comment:
Related to the new plugin system in Apollo Server:
https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md#modified-functionality